server/src/org/apache/cloudstack/region/RegionManagerImpl.java

```diff
@@ -229,7 +232,7 @@ public boolean deleteUser(DeleteUserCmd cmd) {
      */
     @Override
     public Domain updateDomain(UpdateDomainCmd cmd) {
-        return _domainMgr.updateDomain(cmd);
+        return _managementService.updateDomain(cmd);
```

Comment:
Instead of changing the method that is getting called, the missing one in search query can be added here and it seems the updateDomain method in managementserviceimpl is not in use. It can be removed